frontend/src/pages/answer/kgBubble/KgBubble.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable indent, no-use-before-define, func-names, no-return-assign */
```

Comment:
This is an interesting idea. I wonder how it would work with the physics. Would you expect a hovered node to push the surrounding nodes away when it expands, or overlap? And then would it shrink back down if you started to drag it? Opened #189